/builddir/gnome-builder-46.2/src/plugins/xml-pack/ide-xml-parser.c:384:9: error: assigning to 'xmlError *' (aka 'struct _xmlError *') from 'const xmlError *' (aka 'const struct _xmlError *') discards qualifiers [-Werror,-Wincompatible-pointer-types-discards-qualifiers]
  384 |   error = xmlCtxtGetLastError (context);
--
--- a/src/plugins/xml-pack/ide-xml-parser.c
+++ b/src/plugins/xml-pack/ide-xml-parser.c
@@ -362,7 +362,7 @@
 {
   IdeDiagnostic *diagnostic;
   xmlParserCtxt *context;
-  xmlError *error;
+  const xmlError *error;
   const gchar *base;
   const gchar *current;
   g_autofree gchar *msg = NULL;
--- a/src/plugins/xml-pack/ide-xml-validator.c
+++ b/src/plugins/xml-pack/ide-xml-validator.c
@@ -63,7 +63,7 @@
 static IdeDiagnostic *
 create_diagnostic (IdeXmlValidator        *self,
                    GFile                  *file,
-                   xmlError               *error,
+                   const xmlError               *error,
                    IdeDiagnosticSeverity   severity)
 {
   g_autoptr(IdeLocation) loc = NULL;
